tasks.py

```diff
@@ -583,12 +666,34 @@ def pylint(context):
     run_command(context, command)
 
 
-@task
-def pydocstyle(context):
-    """Run pydocstyle to validate docstring formatting adheres to NTC defined standards."""
-    # We exclude the /migrations/ directory since it is autogenerated code
-    command = "pydocstyle ."
-    run_command(context, command)
+@task(aliases=("a",))
+def autoformat(context):
```

Comment:
Really smart and useful addition. It seems though that there's some conflict with the `ruff` formatter: when running `inv ruff` it passes but when executing `inv a` it "fixes" (breaks?) at least one file (`models.py`).

Reply:
Thanks, fixed.

ruff will fix only linter errors with `invoke autoformat` now.